pipeline/scripts/setup-ec2-openvpn/openvpn-down.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/sh
+# Copyright (c) 2007-2008 Roy Marples <roy@marples.name>
+# Amended by Kristian Gunstone for manjaro-openrc
+# Released under the 2-clause BSD license.
```

Comment:
Where have these scripts come from? https://opensource.org/licenses/BSD-2-Clause implies that there should be more to this license statement.

Reply:
I took the files from https://gist.github.com/gammy/b821ff02abfb23d0e41524f96d8a1b0a and https://gist.github.com/gammy/23c071e62f6f93ea8d1b42e0c110d8c9, as mentioned at the bottom of each file.

Looking online a bit more, I think a better source of these files is [https://github.com/OpenRC/openrc/tree/fee2ffe559bc39beec16585daf557b902a53137b/support/openvpn](here). The up.sh/down.sh files there specifically reference the [LICENSE file at the root of that repo](https://github.com/OpenRC/openrc/blob/fee2ffe559bc39beec16585daf557b902a53137b/LICENSE), which is the BSD-2 license.

If this all seems ok with you, I'll switch to using those files instead.

Reply:
Sorry, I missed the notes at the bottom of those files! It sounds like it's worth switching to those files.

Reply:
I've made these changes. I'll test them tomorrow morning.

Reply:
I've tested the changes in the new AWS account we've got, and they work as expected (as long as the VPN connection is setup before running the task, and as long as the up/down bash scripts have LF line endings, not CRLF).